src/utils/format-dates.js

```diff
@@ -40,3 +40,27 @@ export const toDateString = (date, options = defaultOptions) => {
     }
     return newDate.toLocaleString(undefined, options);
 };
+
+export const formatDateToPublishDateFormat = date => {
+    const months = [
+        'Jan',
+        'Feb',
+        'Mar',
+        'Apr',
+        'May',
+        'Jun',
+        'Jul',
+        'Aug',
+        'Sep',
+        'Oct',
+        'Nov',
+        'Dec',
+    ];
+    return (
+        months[date.getMonth() - 1] +
```

Comment:
I believe `getMonth` returns a zero-based month, meaning January is represented as 0 so we don't need to +- 1. I checked out the Mozilla docs [here](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date/getMonth) and also checked in my console that we get the expected month. What did you happen to encounter that required the -1 again?

Reply:
oh man, trust me I saw it returning 8 for August yesterday, I'm not able to reproduce this today. Let me fix and write a test case for this.Thanks for catching this.

Reply:
yeah no worries a test should help but also this is a sneaky one